src/goDocumentation.ts

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+import vscode = require('vscode');
+import cp = require('child_process');
+import path = require('path');
+import { CancellationToken, TextDocumentContentProvider, Uri } from 'vscode';
+import { getBinPath, getToolsEnvVars, byteOffsetAt, getFileArchive } from './util';
+
+const missingToolMsg = 'Missing tool: ';
+
+export function getDocumentation(): void {
+	let gogetdoc = getBinPath('gogetdoc');
+	if (!path.isAbsolute(gogetdoc)) {
+		vscode.window.showErrorMessage(missingToolMsg + 'gogetdoc');
```

Comment:
I am guessing this was picked up from `goDeclaration.ts` file.
When a tool is missing, we usually call the `promptForMissingTool` method that will convey the message and prompt the user to install the tool. 
We will need to do the same here instead of just showing the error message